<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddEditPatient</class>
 <widget class="QDialog" name="AddEditPatient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="currentPatientLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Patient Data this will add or edit</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>371</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="medicineDurationLayout">
    <item>
     <layout class="QHBoxLayout" name="medincineLabels">
      <item>
       <widget class="QLabel" name="medicineLabelAssign">
        <property name="text">
         <string>Medcine to Assign</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="intervalsLabel">
        <property name="text">
         <string>Intervals</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="intervalsInput">
      <item>
       <widget class="QComboBox" name="medicineComboBox">
        <item>
         <property name="text">
          <string>Lean</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="numberIntervals">
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>9</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>11</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>12</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="hrsInterval">
        <item>
         <property name="text">
          <string>Hours</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Days</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>120</y>
     <width>371</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="inputLayout">
    <item>
     <layout class="QVBoxLayout" name="patinetlabels">
      <item>
       <widget class="QLabel" name="firstNameLabel">
        <property name="text">
         <string>First Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lastNameLabel">
        <property name="text">
         <string>Last Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="dobLabel">
        <property name="text">
         <string>DOB:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="genderLabel">
        <property name="text">
         <string>Gender:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="patientInputs">
      <item>
       <widget class="QLineEdit" name="firstNameInput"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lastNameInput"/>
      </item>
      <item>
       <widget class="QLineEdit" name="dobInputMaybe"/>
      </item>
      <item>
       <widget class="QComboBox" name="genderComboBox">
        <item>
         <property name="text">
          <string>Male</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Female</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>371</width>
     <height>226</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableWidget" name="scheduleTableWidget"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="selectionLabels">
      <item>
       <widget class="QPushButton" name="okButton">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="selectedPatientLabel">
      <property name="text">
       <string>Patient:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="selectedPatientCombo"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
